### PR TITLE
docs: [WIP] other way to fix  no-js dark theme 

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [ ] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

fixes https://github.com/eslint/eslint/issues/18076

### CODE REVIEW

File: docs/src/assets/scss/tokens/themes.scss [hunk 1]
```diff
@@ -132,6 +144,10 @@ html[data-theme="light"] {
     --footer-background-color: var(--color-neutral-25);
     --outline-color: var(--color-brand);
     --img-background-color: #fff;
```
Comment (JoshuaKGoldberg) [inline docs/src/assets/scss/tokens/themes.scss:146]: [Non-Actionable] Existing code: this `--img-background-color` is only used in `div.img-container`, which is used in `contribute/architecture` and `extend/code-path-analysis` pages. In light mode the images on those pages have a white background anyway. So 👍 there's no visual difference from manually adding this to the default `:root` styles.

Just noting as it's the only exception to the styles matching up. 🙂

**Review by JoshuaKGoldberg — APPROVED** [review]
This looks great to me, thanks for the detailed explanation & sending this!

I compared the two pairs of styles: `@media`-based and `html-data-theme]`-based. They all have the same attributes except for the existing https://github.com/eslint/eslint/pull/18077/files#r1477122882 that doesn't have any issues associated with it.

💯 !

**Comment by amareshsm** [comment]
> It looks like this PR is preferred over #18071? Can we close #18071 and mark this PR as ready for review?

Yes, #18071 is closed. This is not ready for review some components I have to update I will do this week.

**Comment by nzakas** [comment]
@harish-sethuraman please don't just remove the "stale" label, we need to know why it's not stale. In this case, we should follow up with @amareshsm to see if he's still working on it.

**Comment by amareshsm** [comment]
> @harish-sethuraman please don't just remove the "stale" label, we need to know why it's not stale. In this case, we should follow up with @amareshsm to see if he's still working on it.

Hey Sorry missed this I am working on this. I will complete it in 1 or 2 days.

**Comment by Rec0iL99** [comment]
@amareshsm are you still working on this?

**Pull request closed without merging**